### ISSUE TITLE
Click to keep submenu item open

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#304]
Something that I noticed while working with the import/export menu items.

In other programs, I am used to left-clicking on a menu item that has sub items to keep it open while I select the submenu item.  Our menu does not behave this way, if I click on the menu item with sub menu items, it just closes instead of staying open.

I subconsciously use this feature, it took me a minute to figure what was going on and why it was behaving in this annoying way! :)

**Comment by dondi** [comment]
OK, this one will need further investigation. My first impression is that we are looking at default Bootstrap behavior and so this will be tricky to code around. However I seem to recall that Nicole didn’t go _full_ Bootstrap for some reason; that leaves a small chance that this behavior might be in our code instead. Either way, this will need further code examination before we can determine the level of effort needed.

**Comment by kdahlquist** [comment]
That's fine.  No rush.

**Comment by kdahlquist** [comment]
Backlogging this for future work.
